guides/source/action_controller_overview.md

```diff
@@ -457,7 +457,7 @@ class LoginsController < ApplicationController
   # "Delete" a login, aka "log the user out"
   def destroy
     # Remove the user id from the session
-    @_current_user = session[:current_user_id] = nil
+    session.delete(:current_user_id)
```

Comment:
This would not clear the memoized user in `@_current_user` and could lead to unintended side effects. The user's ID is only retrieved from session once per request in this example so just deleting the session key wouldn't work.